### PR TITLE
start a FAQ

### PR DESCRIPTION
**Pull request by schwabe**

### CODE REVIEW

File: faq.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+Connecting and flashing receivers with newer Q X7 models is a bit
+different from other transmitters see the
+[S-PORT flashing](s-port_flashing.md) section for more details.
+
+
+## Two XJT modules, telemetry from internal and external module, Crossfire and internal module 
+
+This is a bit more exotic question but comes up often enough so that it needs a proper explaination why this problematic.
+
+Smart.port for S.port for short is a bus protocol that is used for
```
Comment (projectkk2glider) [inline faq.md]: Smart.port (S.port for short) is a ...

File: faq.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+everything is okay.
+
+Even binding a receiver without telemetry is not enough to stop a XJT
+module from sending telemetry, as there some module status messages
+that are contantly sent. The only way to ensure that no telemetry is
+coming from an external XJT is though setting the DIP switches.
+
+For the R9M module, enabling/disabling telemetry can be done by
+software. OpenTX will automatically disable telemetry if internal
+module is on too, to avoid these harde to detect errors.
```
Comment (projectkk2glider) [inline faq.md]: harde -> hard

**Review by projectkk2glider — COMMENTED** [review]
Very good. I expect this will expand with time with more questions/answers.

**Comment by schwabe** [comment]
@projectkk2glider thanks for reviewing.

**Pull request merged**